.github/workflows/execute-tests-and-promote.yml

```diff
@@ -145,11 +145,11 @@ jobs:
       matrix:
         test:
           - integration
-          - kat-envoy3-1-of-5
```

Comment:
In general, I'm cool with this but doesn't apro.git depend on this naming convention? If yes, then can we separate this into its own PR because it seems unrelate to removing gold file machinery.

Reply:
Yeah, I did call that out in the PR description. I considered it to be part of the gold machinery because it was there to have both `-envoy` (non-gold) and `-local` (gold) names.  But I can split it in to a separate PR if you really want.

Reply:
OK, I've split it in to two PRs: https://github.com/emissary-ingress/emissary/pull/4875